store/dataStore.ts

```diff
@@ -1,17 +1,41 @@
 import { create } from 'zustand';
+import { produce } from 'immer';
 
-interface Example {
-    id: number;
-    name: string;
+interface State {
```

Comment:
m: Use the Interface's I created for the API. If there isn't one to match the props you have, create it. Keeping the same Interfaces in the same location will

Reply:
✅ Done!